platformio.ini

```diff
@@ -11,6 +11,15 @@
 ;[common]
 ;custom_prog_version = "\"0.4.20\""
 
+[bsc_common]
+build_flags =
+  -std=gnu++2a
+  -std=gnu11
+build_unflags =
+  -std=gnu++11
+  -std=gnu99
+lib_deps =
+  https://github.com/meikjaeckle/ESP32-TWAI#refactor-twai-driver
```

Comment:
Must be switch back to https://github.com/shining-man/ESP32-TWAI after PR https://github.com/shining-man/ESP32-TWAI/pull/1 has been merged.